src/tools/admin/conseillers/update-date-demission.js

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env node
+'use strict';
+require('dotenv').config();
+const cli = require('commander');
+const dayjs = require('dayjs');
+
+const { execute } = require('../../utils');
+
+cli.description('Modification date de démission pour un conseiller')
+.option('-c, --conseiller <id>', 'id: id PG du conseiller')
+.option('-s, --structure <id>', 'id: id PG de la structure')
+.option('-d, --date <date>', 'date : entrer la date de démission du conseiller sous ce format AAAA-MM-DD')
+.helpOption('-e', 'HELP command')
+.parse(process.argv);
+
+const formatDate = date => {
+  return dayjs(date, 'YYYY-MM-DD').toDate();
+};
+
+const regexDateRupture = new RegExp(/^((202)[1-9])(-)(((0)[0-9])|((1)[0-2]))(-)([0-2][0-9]|(3)[0-1])$/);
+
+execute(__filename, async ({ db, logger, exit }) => {
+
+  const idConseiller = ~~cli.conseiller;
+  const idStructure = ~~cli.structure;
```

Comment:
```suggestion
  const idConseiller = +cli.conseiller;
  const idStructure = +cli.structure;
```

Reply:
yes noté de côté, comme ca revient au même